sensor_msgs/Image img
uint16 msg_seq